code/modules/mob/living/carbon/xenomorph/xenoprocs.dm

```diff
@@ -404,7 +404,7 @@
 	return TRUE // normal density flag
 
 /obj/structure/razorwire/acid_spray_act(mob/living/carbon/xenomorph/X)
-	. = ..()
+	take_damage(100, "acid", "acid")
```

Comment:
use defines.

Reply:
Also the damage should be what it was, which is double acid_spray_structure_damage. Not all xenos could actually insta break razor.

Reply:
fixed